dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -163,9 +165,10 @@ private LinkageChecker(
             }
           }
         }
-        if (!classFile.getClassPathEntry().getFileNames()
-            .contains(classSymbol.getClassBinaryName())) {
 
+        String classBinaryName = classSymbol.getClassBinaryName();
+        String classFileName = classDumper.getFileName(classBinaryName);
```

Comment:
I was thinking to rename this variable to `fileName` or `fileNameOfClass`. But `classFileName` fits best.